### ISSUE TITLE
New release version?

### ISSUE THREAD

**Issue by Neustradamus** [serghey-rodin/vesta#2280]
Dear Vesta team, @serghey-rodin,

Last version is 1.0.0-5 (Nov 23, 2021)
- https://github.com/serghey-rodin/vesta/releases
- https://github.com/serghey-rodin/vesta/tags

Soon 1,5 years without update, it is time to do it?

**Comment by manitra** [comment]
You would save energy by switching to https://hestiacp.com/ . VestaCP is **dead**.

**Comment by outrolled** [comment]
Vesta is back under active development and maintenance, however I expect it to take a few months before we get a new version out.

**Comment by Neustradamus** [comment]
Dear all,

Happy New Year 2025!

@outroll team, @outrolled: What is the status one year after?

Thanks in advance.

**Comment by Neustradamus** [comment]
There is a post here, I have done a comment:
- https://github.com/outroll/vesta/discussions/2296

**Comment by mckaygerhard** [comment]
unfortunately if there is already a hestiacp doing all the things vesta do.. so..  the best choice is **a new direction to a full frendly containers based manatgement**.. @outrolled and no.. not choose winbuntu or rpm based.. Debian has a long time support program divided in 3 phases: the normal life cicle, the ExLTS and the Frexian. basically each Debian has at least 10 to 13 years of support for any server focused package..

i mean Just like the Chinese kings do, it is better to attack the market that still needs to be helped than to occupy the saturated market. curerntly OMV (open media vault) made simple plugins for containers .. no orchestation but enought to manage several ones.. if OMV plugin developer can made a simpel plugin inside a NAS system.. why not VEsta base their management in contanires..

NOTE: i m detractor and hater of those crap and shit named containers and dockers ! BUT i m telling you, so far there is no user-friendly system that manages dockers except the plugin for OMV, so you have here a great window to one opportunity @outrolled

**Comment by Neustradamus** [comment]
@outrolled: Have you the date for the new release?

We are now in April 2025.

The latest is always 1.0.0-5 (Nov 23, 2021), more 3 years and 4 months.

A lot of servers have already moved to another panel which is secure instead of unsecure VestaCP (vulnerabilities).

Thanks in advance.

**Comment by Neustradamus** [comment]
@outrolled: Have you progressed on it?

**Comment by outrolled** [comment]
@Neustradamus apologies for the delay getting the new version of Vesta. We are still working on it, but I had to focus on other projects for the last couple of months. The good news is that we hired a new fulltime developer that will dedicate 100% of his time towards Vesta, and we will have 1.5 fulltime developers working on VestaCP for the foreseeable future.
In terms of your compatibility question, we are actually rebuilding the core of VestaCP on Rust (instead of Go), and focusing on having a single binary that runs on almost any OS flavor and architecture (x84,amd64,arm, arm64). This is taking us a lot longer to build that originally expected, but it will provide a memory safe, scalable, enterprise ready platform from day 1.

**Comment by Neustradamus** [comment]
@outrolled: Thanks for your answer.

There will be an upgrade system?

But the current problem is that CVEs are not fixed...

**Comment by outrolled** [comment]
@Neustradamus I agree that is the main issue - the goal was for the new version to automatically migrate users, so the existing CVEs would be resolved all at once. Given that the new version is still some time away, it may be worth to push an interim update. It would be great if the community could help with this one, so that we cut the vulnerabilities asap.

**Comment by Neustradamus** [comment]
@outrolled: Have you progressed?

**Comment by HHawk** [comment]
VestaCP is dead...
HestiaCP is dead...

Tired of waiting. Migrated our last servers. Didn't.ind to pay, but waiting for months is not a solution.

**Comment by mckaygerhard** [comment]
@HHawk  i migrated to https://myvestacp.com/ cos is basically compatible.. DEbian powered as must be

**Comment by Neustradamus** [comment]
Dear @outroll team, @outrolled,

It is the end of the year 2025, what is the progress to solve CVEs, etc.?

Thanks in advance.

Linked to:
- https://github.com/outroll/vesta/issues/2280
- https://github.com/outroll/vesta/discussions/2296
- https://github.com/outroll/vesta/discussions/2298
- https://github.com/outroll/vesta/discussions/2299
- https://github.com/outroll/vesta/discussions/2300
- https://github.com/outroll/vesta/issues/2312
- https://github.com/outroll/vesta/discussions/2315
- https://github.com/outroll/vesta/issues/2321

**Comment by Neustradamus** [comment]
Dear all,

@Dennis-SEG has done a very good job here:
- https://github.com/outroll/vesta/pull/2322

What do you think?

**Comment by Neustradamus** [comment]
Dear @outroll team, @outrolled,

Have you progressed on it?

Thanks in advance.
